Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCellView_Tests.swift

```diff
@@ -0,0 +1,168 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerMentionCellView_Tests: XCTestCase {
+    /// Default reference width for the cell. Not related to any screen size.
+    private static var defaultCellWidth: CGFloat = 300
+    /// To simulate cell behavior with autoresizing height according to content
+    /// we create this height value as well.
+    private static var defaultCellHeight: CGFloat = 50
+
+    private var chatUserOffline: _ChatUser<NoExtraData>!
+    private var chatUserOnline: _ChatUser<NoExtraData>!
+    private var chatUserNoName: _ChatUser<NoExtraData>!
+
+    override func setUp() {
+        super.setUp()
+        chatUserOffline = .mock(
+            id: "darkside37",
+            name: "Mr Vader",
+            imageURL: TestImages.vader.url,
+            isOnline: false
+        )
+
+        chatUserOnline = .mock(
+            id: "darkside37",
+            name: "Mr Vader",
+            imageURL: TestImages.vader.url,
+            isOnline: true
+        )
+
+        chatUserNoName = .mock(
+            id: "yoda",
+            imageURL: TestImages.yoda.url,
+            isOnline: true
+        )
+    }
+
+    func test_emptyAppearance() {
+        let view = ChatMessageComposerMentionCellView().withoutAutoresizingMaskConstraints
+        view.widthAnchor.constraint(equalToConstant: Self.defaultCellWidth).isActive = true
+        view.heightAnchor.constraint(greaterThanOrEqualToConstant: Self.defaultCellHeight).isActive = true
+
+        AssertSnapshot(view)
```

Comment:
I still think we should be using the size argument of the `AssertSnapshot(size:)`. We are basically repeating what the framework is already doing inside. Also, defining the constraints is more extensive than just passing a CGSize. WDYT @VojtaStavik ? Either way, I think we should try to reuse this since this will be very repetitive, but then again, I think this is already the aim of the `size` property. But if for some reason we still want to add our own constraints, it should happen inside the `AssertSnapshot` so we can reuse this.

Reply:
-> slack

Reply:
This is fixed with https://github.com/GetStream/stream-chat-swift/pull/819/commits/768c6431fd102be9480ee6cea809c5e353a55c01 after our Slack conversation)